third_party/rust/aes/v0_8/BUILD.gn

```diff
@@ -54,7 +54,10 @@ cargo_crate("lib") {
     "//brave/third_party/rust/cfg_if/v1:lib",
     "//brave/third_party/rust/cipher/v0_4:lib",
   ]
-  if (target_cpu == "x64" || target_cpu == "x86") {
+  if (target_cpu == "arm64" || target_cpu == "x64" || target_cpu == "x86") {
```

Comment:
these changes don't look right to me, why are we having all these changes related to cpufeatures if cpufeatures hasn't changed versions?

Reply:
ok, looks like this is what it was originally supposed to be and a new version of gnrt fixed it?

Reply:
The output for these conditionals changed dramatically and was a bit tricky, and it required updating `gnrt_config.toml` to react to the new set of defaults.